docs/content/Getting-Started/user-guide.md

```diff
@@ -1,11 +1,104 @@
+
+A deployment of KubeStellar MCCM supports multiple clients. Each owns some Workload Execution Clusters (WECs), which are Kubernetes clusters of whatever sort the client wants. A client wants to manage the configuration of the Kubernetes API objects in the WECs from a central place. A client has an inventory of WECs, and manages one or more inventory spaces (ISes) to hold that inventory. The inventory holds a pair of kube-style API objects per WEC, of kind `SyncTarget` and `Location`. A client has one or more workloads that the client wants to be run on the WECs. KubeStellar MCCM does not have an opinion on how a client divides its objects between "workloads"; the MCCM interfaces deal directly with kube-style API objects and we use the term "workload" only informally in documentation. A client maintains its workload descriptions in Workload Description Spaces (WDSes). A client maintains just one copy of each workload object in the core (the MCCM core is the central place where the clients interact with MCCM; it includes the ISes and WDSes). To convey its wishes about what goes where, a client maintains control objects of kind `EdgePlacement` in the WDSes. Each `EdgePlacement` object binds a selection of WECs with both (a) a definition of which workload objects are downsynced and (b) a definition of which workload objects are upsynced. The MCCM implementation implements the prescribed behavior.
+
+In KubeStellar MCCM there are downsynced objects and upsynced objects. A downsynced object is one that a client creates in a WDS. Its desired state propagates from WDS to WEC, and its reported state propagates back towards the core. An upsynced object is created in a WEC and MCCM transports it back towards the core.
+
+For each WEC there is a setup step that establishes its relationship with the core. This is called "WEC bootstrapping".
+
+KubeStellar MCCM uses the APIExport/APIBinding technology of kcp to separate consumers and the provider of MCCM functionality and the uses and definition of the Kubernetes resources for the management of containerized workloads.
+
+KubeStellar MCCM contemplates two classes of users: those who create/admin a deployment of KubeStellar MCCM, and those who use an existing deployment. Naturally, a deployment of KS MCCM has to be created before it can be used.
+
+## Pre-requisites for using KS MCCM
+
+First see the pre-requisites section of [the quickstart](../quickstart/) or of [the extended example](../../Coding%20Milestones/PoC2023q1/example1/).
```

Comment:
Keep it simple - link a single prerequisite section preferably one that is common to the user guide, the quick start, and the extended sample.

Reply:
The prerequisites do have some differences depending on your use case. I think it breaks out like this:

- Are you going to be using an existing deployment of Kubestellar MCCM (user)?
- Are you going to be deploying Kubestellar MCCM and then using it (deployer)?
- Are you going to be playing with the innards of Kubestellar beyond basic deployment configuration (developer)?

I think the existing pre-reqs include file is for the essentials for the user case, right @MikeSpreitzer ?